.github/workflows/main.yml

```diff
@@ -1,15 +1,68 @@
-name: Ubuntu
+name: CI
 
-on: [push]
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
 
 jobs:
-  build:
-
-    runs-on: ubuntu-18.04
+  test:
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6]
+        python-version: [pypy3, 2.7, 3.5, 3.6, 3.7, 3.8]
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        include:
+          - os: ubuntu-latest
+            python-version: pypy2
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout Code
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install coveralls setuptools tox wheel
+
+    - name: Build Wheel
+      run: python setup.py bdist_wheel
+
+    - name: Install Vulture from Wheel (For Linux and MacOS)
+      if: ${{ matrix.os != 'windows-latest' }}
+      run: "python -m pip install --only-binary=:all: ./dist/vulture-*.whl"
+
+    - name: Install Vulture from Wheel (For Windows)
+      if: ${{ matrix.os == 'windows-latest' }}
+      run: "python -m pip install --only-binary=:all: --no-index --ignore-installed --find-links=dist/ vulture"
```

Comment:
Can we use the same way of installing the wheel on all OSs?

Reply:
Yes, we should do that. But I _think_ it broke the build when we last tried doing that (circa: 2017). Let me take a stab at it again!

But, let's try that in a different PR?

Reply:
Whatever you prefer.